### PR TITLE
Updates Wiki page

### PR DESCRIPTION
**Pull request by Rafase282**
1. Add guide about Wiki's Labels.
2. Add other links to guides.
3. Restructures and provides a working table of content.
4. Adds intro to the wiki and removes separated on how to use the wiki.
5. witches the deleted article in the side bar for the `Wiki.md`.
6. Changes all links to be relative ones in the side bar instead of the mix of absolute and relative ones.

This is to be merged with separated commits, do not squash, this is to make reverting some parts easier without reverting everything at once.

_**While this does get added to the wiki, it is still under development and the whole idea is to make all wiki related information easily accessible in an organized matter.**_

### CODE REVIEW

File: Wiki.md [hunk 1]
```diff
@@ -4,17 +4,37 @@ The title will have to change to something better. This will be the central area
...
+  Finds related materials to that topic, usually a page from the [wiki](https://github.com/FreeCodeCamp/freecodecamp/wiki). Examples: `help css` `help bootstrap`
+
+- **`topics`**
+
+  Shows the list of topics.
+
+- **`find $topic`**
 
-How to contribute via PR, helping mod the PR, creating issues, etc.
+  This will search for entries that include `$topic` in the title. Examples: `find wiki`
```
Comment (alayek) [inline Wiki.md]: ~~Examples~~ Example

File: Wiki.md [hunk 2]
```diff
@@ -28,21 +48,37 @@ We currently have a couple of guides to help you contribute, via the browser, co
...
-### How to Become an Issue Moderator for the Wiki
+### Wiki Mods
 
 If you are really committed to contributing to the wiki and you would like to join the team and be able to merge pull request, create new labels, assign mods to specific pull request and more then just check the [requirements to become a wiki mod](Wiki-Become-A-Wiki-Mod) and follow the instructions.
 
-**This page is still under construction, feel free to contribute to it by linking existing resources or extending on the content of this page.**
+## Templates
 
-## Article Templates
+We have a few template for different aspects of the wiki. such as different articles and fixed information to help contribute.
```
Comment (alayek) [inline Wiki.md]: > We have a few ~~template~~ templates for different aspects

Comment (alayek) [inline Wiki.md]: > ~~such~~Such as different articles

Comment (alayek) [inline Wiki.md]: I think you can merge these two sentences together with a comma

File: Wiki.md [hunk 3]
```diff
@@ -28,21 +48,37 @@ We currently have a couple of guides to help you contribute, via the browser, co
...
+### Article Templates
+
+These kind of templates are a guide to the creation of new articles such as algorithms challenges, and informational articles.
 
 - [JavaScript Articles](Wiki-Template-JavaScript)
 - [Challenge Solutions](Wiki-Template-Challenge-Solution)
+
+### Contribution Templates
+
+These templates are visible when you are trying to contribute to the Wiki. They will provide with useful information, guides, and guidelines that you must follow to ensure a proper contribution.
```
Comment (alayek) [inline Wiki.md]:
```diff
- They will provide with useful information
+ They will provide you with useful information
```

**Comment by alayek** [comment]
@Rafase282 great job! Left a few comments here and there. Just fix them and it's LGTM from my side :+1:

**Comment by alayek** [comment]
# LGTM :+1: 

---

**Comment by theflametrooper** [comment]
Thanks @Rafase282 !!

**Pull request merged**